src/app/search/search.component.ts

```diff
@@ -180,6 +180,9 @@ export class SearchComponent implements OnInit, OnDestroy {
     if (!paramMap) {
       return;
     }
+    // Spread operator for assigning the initialAdvancedSearchObject must be used due to an issue where overriding
+    // any of the object's field will result in an error for localhost running without production mode on.
+    // This issue does not appear on dev.dockstore.net.
```

Comment:
Pretty sure the only difference is prod mode vs. non-prod mode. You reproduced it locally with prod mode, didn't you?

If so, you could simplify this whole comment, not mentioning localhost nor dev, and just focusing on prod vs. non-prod mode.

Reply:
Simplified my comment